.github/workflows/ci.yml

```diff
@@ -27,8 +27,10 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          pip install -r tests/requirements.txt
-          pip install -e .
+          pip install -U pip
+          pip install -U wheel
```

Comment:
Always safer to update `pip` and `wheel` to support latest pip flags and have latest & fixed dependency resolution algorithm.